src/Data/NormalKey.purs

```diff
@@ -101,6 +97,8 @@ instance ordNormalKey :: Ord NormalKey where
   compare (NormalKey "Shift") _ = LT
   compare _ (NormalKey "Shift") = GT
   compare (NormalKey "Enter") (NormalKey "Enter") = EQ
+  compare _ (NormalKey "Enter") = GT
+  compare (NormalKey "Enter") _ = LT
   compare (NormalKey x) (NormalKey y) = compare x y
```

Comment:
Anyway, if it seems to difficult, feel free to ignore my suggestions here :) I'm just mentioning ways of creating an instance whose correctness is easy to verify, since I don't trust myself to make that determination for the current code sans property tests (though it is probably correct).